### DISCUSSION TITLE
Add fish barrel inventory count overlay

### DISCUSSION THREAD

**Discussion by jeigarro** [runelite/runelite#16407] [Ideas]
Take the fish barrel plugin found here:
https://github.com/molo-pl/runelite-plugins/tree/fish-barrel

And add an overlay counter that tracks how many fish are inside the fish barrel, similar to the inventory counter found here:
https://github.com/robrichardson13/inventory-count/tree/a9eb8fb64b866751de35ac4614f74b513df038b7

Here is an example of where the overlay would reside:
![image](https://user-images.githubusercontent.com/43282067/227095062-0fb31b01-aba2-4161-9aa2-419afcf893e7.png)

**Comment by Nightfirecat** [comment]
Unless we add fish barrel counting in general in the main client, we will not be adding an infobox for it. You can request the author of that plugin adds this feature instead.
